src/app/pages/model-3d/model-3d-list/model-3d-list.page.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Component,
+  OnInit,
+  inject,
+  ChangeDetectionStrategy,
+  effect,
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+  Validators,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatSelectModule } from '@angular/material/select';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIcon } from '@angular/material/icon';
+import { TranslatePipe } from '@ngx-translate/core';
+import { GeneralContainerComponent } from '@components/layout';
+import { GlbViewerComponent } from '@components/model3D';
+import { AlertTypeEnum } from '@common/enums';
+import { AlertService, FormsService } from '@services/general';
+import { PagesUrlsEnum } from '@common/enums';
+import { Model3DCreateHttpService } from '@services/http';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-model-3d-list',
+  templateUrl: './model-3d-list.page.html',
+  styleUrls: ['./model-3d-list.page.scss'],
+  standalone: true,
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
utilizar el de constants


```suggestion
  changeDetection: Constants.changeDetectionStrategy
```